.env.example

```diff
@@ -10,3 +10,6 @@ KONG_CLIENT_VERIFY_CERTIFICATES=True
 KONG_CLIENT_TEST_MODE=False
 KONG_CLIENT_TEST_FILENAME=records.json
 PLATFORMSH_CLI_TOKEN=your-api-token-here
+API_CLIENT_NAME_PREFIX=
```

Comment:
Could we pre-populate this example with the prefix we are using?

Reply:
Yep - probably worth doing a separate MR for that to avoid confusion. I'll put another one in for it.